prow/pod-utils/decorate/podspec.go

```diff
@@ -815,6 +816,20 @@ func decorate(spec *coreapi.PodSpec, pj *prowapi.ProwJob, rawEnv map[string]stri
 		spec.Volumes = append(spec.Volumes, append(cloneVolumes, codeVolume)...)
 	}
 
+	if pj.Spec.DecorationConfig != nil && pj.Spec.DecorationConfig.SetLimitEqualsMemoryRequest != nil && *pj.Spec.DecorationConfig.SetLimitEqualsMemoryRequest {
```

Comment:
`pj.Spec.DecorationConfig != nil` can be assumed here. It doesn't make a difference though.